app/ide-desktop/lib/dashboard/src/layouts/dashboard/Chat.tsx

```diff
@@ -748,7 +748,6 @@ export default function Chat(props: ChatProps) {
           <textarea
             ref={messageInputRef}
             rows={1}
-            autoFocus
```

Comment:
@somebody1234 Will this change cause any problem for the dashboard? Starting the app with the chat widget's input field focused is interfering with the graph editor's focus logic.

Reply:
i don't think there should be any major issues, but i think it'd be preferred that the `autoFocus` stays and instead the graph editor would grab back focus when it starts - because there are other situations where `autoFocus` is desirable:
- opening chat when the editor is not open
- opening chat *after* the graph editor is opened

Reply:
But, [technically](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus), autofocus works only on page load, doesn't it?